docs/core/tutorials/nft-tutorials/whitelists.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Merkle Tree Whitelist
+displayed_sidebar: coreSidebar
+---
+
+# Merkle Tree NFT Whitelist on CoreSpace using Hardhat
+
```

Comment:
An overall introduction is required at the beginning of the article.

Reply:
Current version is not enough. I think the content of the conclusion can be partially or all moved to the top to show why and when they need to use the merkel tree whitelist.

Reply:
This statement 
```
Storing a whitelist with thousands of addresses in a smart contract would incur high gas fees for each update. However, since in the Merkle Tree verification process, leaf nodes and proofs can be stored off-chain while only the root value needs to be stored on-chain, this method significantly saves on gas costs.
``` 
is not techinically correct. In core space, if you stores a value in one transaction and delete it in another, the gas cost will be far more cheap than that in ethereum thanks to storage collateral

Reply:
The statement doesn't compare with Ethereum, but rather with the Merkle tree method. In Core Space, if the number of whitelisted addresses grows from 1,000 to 9,000, it will still consume a lot of gas. However, if the Merkle tree method is used, it only requires updating the root value in the contract.

Reply:
I think I've misunderstood your original meaning for the `each update` expression and then there is no technical problem. I will later provide a more accurate version